### PR TITLE
manifest: Update mpsl and softdevice controller

### PR DESCRIPTION
**Pull request by martintv**
This fixes an issue with event pooling.

Signed-off-by: Martin Tverdal <martin.tverdal@nordicsemi.no>

### CODE REVIEW

**Review by rugeGerritsen — APPROVED** [review]

**Review by hermabe — APPROVED** [review]

**Comment by rlubos** [comment]
This needs a rebase, nrfxlib SHA got just updated on main.

**Comment by martintv** [comment]
> This needs a rebase, nrfxlib SHA got just updated on main.

yes, done.

File: west.yml [hunk 1]
```diff
@@ -116,7 +116,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 5b63a4c8939ca6b534ac983bd513d356c9a55784
+      revision: pull/876/head
```
Comment (rlubos) [inline west.yml:119]: @martintv Please update SHA `c9c226c90f1bf19e2ce9ee9e85e1b621d09db8d5`

Comment (martintv) [reply west.yml]: done!

**Review by rlubos — APPROVED** [review]

**Pull request merged**